### PR TITLE
Maintenence Update

### PR DESCRIPTION
**Pull request by ivacf**
Read commit messages for more detail about changes

### CODE REVIEW

File: app/src/commonTest/java/uk/co/ribot/androidboilerplate/test/common/TestDataFactory.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * Factory class that makes instances of data models with random field values.
+ * The aim of this class is to help setting up test fixtures.
+ */
+public class TestDataFactory {
+
+    public static String randomUuid() {
+        return UUID.randomUUID().toString();
+    }
+
+    public static Ribot makeRibot() {
```
Comment (hitherejoe) [inline app/src/commonTest/java/uk/co/ribot/androidboilerplate/test/common/TestDataFactory.java]: makes more sense called this now

File: app/src/main/java/uk/co/ribot/androidboilerplate/data/model/Name.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Name name = (Name) o;
+
+        if (first != null ? !first.equals(name.first) : name.first != null) return false;
+        return !(last != null ? !last.equals(name.last) : name.last != null);
+
```
Comment (hitherejoe) [inline app/src/main/java/uk/co/ribot/androidboilerplate/data/model/Name.java]: space!

File: app/src/main/java/uk/co/ribot/androidboilerplate/data/model/Profile.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        if (email != null ? !email.equals(profile.email) : profile.email != null) return false;
+        if (hexColor != null ? !hexColor.equals(profile.hexColor) : profile.hexColor != null)
+            return false;
+        if (avatar != null ? !avatar.equals(profile.avatar) : profile.avatar != null)
+            return false;
+        if (dateOfBirth != null
+                ? !dateOfBirth.equals(profile.dateOfBirth) : profile.dateOfBirth != null)
+            return false;
+        return !(bio != null ? !bio.equals(profile.bio) : profile.bio != null);
+
```
Comment (hitherejoe) [inline app/src/main/java/uk/co/ribot/androidboilerplate/data/model/Profile.java]: and again!

File: app/src/main/java/uk/co/ribot/androidboilerplate/data/model/Name.java (hunk 2 above)
Comment (ivacf) [inline app/src/main/java/uk/co/ribot/androidboilerplate/data/model/Name.java]: Android studio automatically generates that bit of code with that space, I'm not sure why. I don't think it matters too much... Same for the one below.

**Pull request merged**